src/main.js

```diff
@@ -431,46 +380,34 @@ let load = base.loader({
             new Date(), now);
       let count = await TaskGroupMember.expire(now);
       debug("Expired %s task-group members", count);
-
-      // Stop recording statistics and send any stats that we have
-      base.stats.stopProcessUsageReporting();
-      return influx.close();
     }
   },
 
   // Create the task-dependency expiration process (periodic job)
   'expire-task-dependency': {
-    requires: ['cfg', 'TaskDependency', 'monitor', 'influx'],
-    setup: async ({cfg, TaskDependency, influx}) => {
+    requires: ['cfg', 'TaskDependency', 'monitor'],
+    setup: async ({cfg, TaskDependency, monitor}) => {
       var now = taskcluster.fromNow(cfg.app.taskExpirationDelay);
       assert(!_.isNaN(now), "Can't have NaN as now");
 
       // Expire task-dependency using delay
       debug("Expiring task-dependency at: %s, from before %s", new Date(), now);
       let count = await TaskDependency.expire(now);
       debug("Expired %s task-dependency", count);
-
-      // Stop recording statistics and send any stats that we have
-      base.stats.stopProcessUsageReporting();
-      return influx.close();
     }
   },
 
    // Create the task-requirement expiration process (periodic job)
   'expire-task-requirement': {
-    requires: ['cfg', 'TaskRequirement', 'monitor', 'influx'],
-    setup: async ({cfg, TaskRequirement, influx}) => {
+    requires: ['cfg', 'TaskRequirement', 'monitor'],
+    setup: async ({cfg, TaskRequirement, monitor}) => {
       var now = taskcluster.fromNow(cfg.app.taskExpirationDelay);
       assert(!_.isNaN(now), "Can't have NaN as now");
 
       // Expire task-requirement using delay
       debug("Expiring task-requirement at: %s, from before %s", new Date(), now);
       let count = await TaskRequirement.expire(now);
       debug("Expired %s task-requirement", count);
-
-      // Stop recording statistics and send any stats that we have
-      base.stats.stopProcessUsageReporting();
```

Comment:
So in the old days did this some sort of CPU/memory monitoring... do we do any of that now?
If so, then don't we need to give it like a process identifier?
If not... then we probably shouldn't require `monitor` here, and add it once we have some sort of CPU/memory monitoring again...